dictionaries/ar.json

```
@@ -462,6 +467,45 @@
       "viewProducts": "عرض المنتجات",
       "clientsSection": "العملاء",
       "clientsDescription": "المستخدمون المعينون لهذا العمل",
+      "clientDetailsTitle": "تفاصيل العميل",
```

Biome (2.4.10)

[error] The key clientDetailsTitle was already declared.

(lint/suspicious/noDuplicateObjectKeys)